apps/src/code-studio/pd/professional_learning_landing/upcomingWorkshops.jsx

```diff
@@ -23,6 +24,38 @@ const UpcomingWorkshopsTable = React.createClass({
     workshops: React.PropTypes.arrayOf(workshopShape)
   },
 
+  getInitialState() {
+    return {
+      showCancelModal: false,
+      cancelledEnrollmentCode: undefined
```

Comment:
nit: this name implies that it has already been canceled. How about `enrollmentCodeToCancel`, or `cancellingEnrollmentCode`?